lib/features/base/action/ui_action.dart

```diff
@@ -7,11 +7,9 @@ abstract class Action with EquatableMixin {}
 
 abstract class UIAction extends Action {}
 
-abstract class FcmAction extends Action {}
-
-abstract class FcmStateChangeAction extends FcmAction {
+abstract class PushNotificationStateChangeAction extends Action {
   final TypeName typeName;
   final jmap.State newState;
 
-  FcmStateChangeAction(this.typeName, this.newState);
+  PushNotificationStateChangeAction(this.typeName, this.newState);
```

Comment:
Do we confirm FCM is kept for mobile and websocket only used for WEB?

Reply:
Yes confirmed

Reply:
We should check the FCM stream on mobile. (Attach demo video)

Reply:
https://github.com/user-attachments/assets/4a2a4d65-72f3-4fdc-bdc3-e388fa352564